packages/editor/src/client/atoms/editorTemplate.tsx

```diff
@@ -22,7 +22,7 @@ export function EditorTemplate({children, navigationChildren}: EditorTemplatePro
           display: 'flex',
           position: 'sticky',
           top: 0,
-          zIndex: 3,
+          zIndex: 10,
```

Comment:
What browser is it you use where it overlaps? It doesn't overlap on my screen.

Reply:
I use both google chrome and Microsoft edge and for me it overlapped on both.

Reply:
Oh I see it now. Good catch :+1: